test/lib/models/currency.js

```diff
@@ -22,7 +22,8 @@ describe('Currency model', () => {
       id: new Array(...(new Array(5))).map(() => Math.random()),
       name: VALID_STRINGS,
       pool: VALID_STRINGS,
-      explorer: VALID_STRINGS
+      explorer: new Array(...(new Array(5))).map(() => []),
+      walletFx: new Array(...(new Array(5))).map(() => [])
```

Comment:
Im not quite I understand the use case of this model. What request should this be used for?

Is it `https://api-pub.bitfinex.com/v2/conf/pub:map:currency:wfx`?

Reply:
Yes, is for `pub:map:currency:wfx` wfx, stands for walletFx
This how a response would look like: 
`[{"_events":{},"_eventsCount":0,"_fields":{"id":0,"name":1,"pool":2,"explorer":3,"symbol":4,"walletFx":5},"_boolFields":[],"id":"BTC","name":"Bitcoin","pool":null,"explorer":["https://blockstream.info","https://blockstream.info/address/VAL","https://blockstream.info/tx/VAL"],"symbol":"BTC","walletFx":[["LBT",1]]},{"_events":{},"_eventsCount":0,"_fields":{"id":0,"name":1,"pool":2,"explorer":3,"symbol":4,"walletFx":5},"_boolFields":[],"id":"LBT","name":"Bitcoin (Liquid)","pool":null,"explorer":["https://blockstream.info/liquid","https://blockstream.info/liquid/address/VAL","https://blockstream.info/liquid/tx/VAL"],"symbol":"LBTC","walletFx":[["BTC",1]]},{"_events":{},"_eventsCount":0,"_fields":{"id":0,"name":1,"pool":2,"explorer":3,"symbol":4,"walletFx":5},"_boolFields":[],"id":"LET","name":"Unus Sed LEO (ERC20)","pool":"ETH","explorer":["https://etherscan.io","https://etherscan.io/address/VAL","https://etherscan.io/tx/VAL"],"symbol":"LEO-ERC20","walletFx":[["LEO",1]]}]`
walletFx are the equivalent currencies for a currency. So For example LET is equivalent to LEO.
This is required for Reports, as when doing Tax Reports we cant convert the USD equivalent of a LET deposit as the Pair tLETUSD does not exist, so what we should do is check the pair tLEOUSD and for that we need to know that LET is equivalent to LEO.